<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateWindow</class>
 <widget class="QMainWindow" name="UpdateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firmware Flash Tool V1.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="McuBox">
        <property name="maximumSize">
         <size>
          <width>87</width>
          <height>22</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MMCU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SMCU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CMCU</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="BaudBox">
        <property name="maximumSize">
         <size>
          <width>87</width>
          <height>22</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="PortBox">
        <property name="maximumSize">
         <size>
          <width>87</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Open_com">
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Update">
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>更新</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Open_file">
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>打开文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="textoutput"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Language">
    <property name="title">
     <string>语言（Language）</string>
    </property>
    <addaction name="action_Chinese"/>
    <addaction name="action_English"/>
   </widget>
   <addaction name="menu_Language"/>
  </widget>
  <action name="action_Chinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中文（Chinese）</string>
   </property>
  </action>
  <action name="action_English">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>英文（English）</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
